.github/scripts/builder-compile.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash
+
+
+do_patch () {
+    PATCH_DIR=$1
+    PATCH_SRC=$2
+    if [ -d "${PATCH_DIR}" ]
+    then
+        for patch in $(find ${PATCH_DIR} -type f -name "*.patch"); do
+            echo Applying patch ${patch}
+            patch -f -d "${PATCH_SRC}" -p1 < $patch
+        done;
+    fi
+}
+
+WORKING=`pwd`
+OUTDIR=${2:-$WORKING}
+
+wget ${1:-http://github.com/apache/qpid-proton/archive/main.tar.gz} -O qpid-proton.tar.gz
+
+mkdir -p qpid-proton-src build staging proton_build proton_install
+tar -zxf qpid-proton.tar.gz -C qpid-proton-src --strip-components 1
+
+do_patch "patches/proton" qpid-proton-src
+
+cd proton_build
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_CPP=OFF -DBUILD_RUBY=OFF -DBUILD_GO=OFF -DCMAKE_INSTALL_PREFIX=/usr -DSYSINSTALL_PYTHON=ON -DBUILD_TLS=ON $WORKING/qpid-proton-src/ \
```

Comment:
`-DBUILD_BINDINGS=python` works better; say what you want

Reply:
... then I can get rid of these, right: `-DBUILD_CPP=OFF -DBUILD_RUBY=OFF -DBUILD_GO=OFF` ?

Reply:
yes, that is the idea; GitHub CI does it this way too, https://github.com/skupperproject/skupper-router/blob/main/.github/workflows/build.yaml#L42-L49

Reply:
I have also replaced `-DBUILD_TLS=ON` with `-DSSL_IMPL=openssl` as the former is no longer a valid option.